reconcile/utils/output.py

```diff
@@ -13,7 +13,7 @@ def print_output(
     options: Mapping[str, Union[str, bool]],
     content: list[dict],
     columns: Iterable[str] = (),
-):
+) -> Any:
```

Comment:
from the code below, this function can return `str | None`, any other possible types?

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4355/commits/efed676f5138760f17d7ffa1b97b7c205b260eaf